test/integration/src/test/java/io/pravega/test/integration/MetricsTest.java

```diff
@@ -199,18 +202,20 @@ public void metricsTimeBasedCacheEvictionTest() throws Exception {
 
             readAllEvents(reader1);
 
+            final String[] streamTags = segmentTags(scope + "/" + STREAM_NAME + "/0.#epoch.0");
             AssertExtensions.assertEventuallyEquals(bytesWritten, () -> {
-                return (long) MetricRegistryUtils.getCounter("pravega.segmentstore.segment.read_bytes", segmentTags(scope + "/" + STREAM_NAME + "/0.#epoch.0")).count();
+                return MetricRegistryUtils.getCounter(SEGMENT_READ_BYTES, streamTags) == null
+                        ? -1 : (long) MetricRegistryUtils.getCounter(SEGMENT_READ_BYTES, streamTags).count();
             }, 10000);
 
             String readerGroupName2 = readerGroupName + "2";
             log.info("Creating Reader group : {}", readerGroupName2);
 
             readerGroupManager.createReaderGroup(readerGroupName2,
-                                                 ReaderGroupConfig.builder()
-                                                                  .stream(Stream.of(scope, STREAM_NAME))
-                                                                  .automaticCheckpointIntervalMillis(2000)
-                                                                  .build());
+                    ReaderGroupConfig.builder()
```

Comment:
Same here.

Reply:
The continuation indent should be 8 only.